syntax = "proto3";

message Empty {
}

message Statement {
  string type = 1;
}

message Entity {
  string type = 1;
  repeated string values = 2;
}

message EntityResponse {
  repeated Entity entities = 1;
  string label = 2;
}

message Command {
   string type = 1;
}

message Accuracy {
  float acc = 1;
}

service MainKG {
  rpc RequestKeywords(Empty) returns (Statement);
  rpc ReceiveKeywords(EntityResponse) returns (Empty);
}
